bot/code_review_bot/workflow.py

```diff
@@ -103,6 +102,47 @@ def run(self, revision):
 
         return issues
 
+    def ingest_autoland(self, revision):
+        """
+        Simpler workflow to ingest autoland revision
+        """
+        assert revision.repository == REPO_AUTOLAND, "Need an autoland revision"
+        logger.info(
+            "Starting autoland ingestion",
+            revision=revision.id,
+            bugzilla=revision.bugzilla_id,
+            title=revision.title,
+            mercurial_revision=revision.mercurial_revision,
+        )
+
+        assert (
+            self.backend_api.enabled
+        ), "Backend storage is disabled, no autoland ingestion possible"
+
+        # Find potential issues in the autoland task group
+        # There should be no issues, so the build is OK
+        tasks = self.queue_service.listTaskGroup(settings.autoland_group_id)
```

Comment:
Don't you need the continuationToken dance in order to get all tasks?

Reply:
I now use the official pagination support